tools/rna_tools/sortmerna/test-data/test2_log.txt

```diff
@@ -1,34 +0,0 @@
- Sat Sep 23 09:30:45 2017
```

Comment:
log becomes empty?

Reply:
This option has been removed from the wrapper.